specs/posix/socket_events.table

```diff
@@ -1,7 +1,7 @@
 table_name("socket_events")
-description("Track network socket opens and closes.")
+description("Track network socket bind, connect, and accepts.")
 schema([
-    Column("action", TEXT, "The socket action (bind, listen, close)"),
```

Comment:
I couldn't find any evidence in the existing code of handling for `listen` or `close`. I did find `connect` on both macOS and Linux, and `accept` on Linux.